src/components/queryBuilder/QueryBuilder.tsx

```diff
@@ -197,8 +206,27 @@ export const QueryBuilder = (props: QueryBuilderProps) => {
           timeFieldType={builder.timeFieldType}
           onTimeFieldChange={onTimeFieldChange}
           fieldsList={fieldsList}
+          timeFieldTypeCheckFn={isDateType}
+          labelAndTooltip={selectors.components.QueryEditor.QueryBuilder.TIME_FIELD}
         />
       )}
+      {
+        // Time and LogLevel fields selection for Logs Volume histogram in the Explore mode
+        // TODO: show only in the Explore mode
```

Comment:
Do we have this information available within the plugin?

Reply:
Found it. `CoreApp` is available from `CHQueryEditorProps` in `CHQueryEditor` and could be passed to `QueryBuilder`